llvm/test/CodeGen/PowerPC/gcov_ctr_ref_init.ll

```diff
@@ -0,0 +1,129 @@
+; Tests if the __llvm_gcov_ctr section contains a .ref pseudo-op
+; referring to the __llvm_covinit section.
+; RUN: llc < %s | FileCheck --check-prefixes=CHECK,CHECK-RW %s
+; RUN: llc -mxcoff-roptr < %s | FileCheck --check-prefixes=CHECK,CHECK-RO %s
+
+target datalayout = "E-m:a-p:32:32-Fi32-i64:64-n32"
+target triple = "powerpc-ibm-aix"
+
+; CHECK-RW: .csect __llvm_covinit[RW],3
+; CHECK-RO: .csect __llvm_covinit[RO],3
+; CHECK:         .vbyte  4, __llvm_gcov_writeout[DS]
+; CHECK-NEXT:    .vbyte  4, __llvm_gcov_reset[DS]
+; CHECK: __llvm_gcov_ctr.1:
+; CHECK-NEXT:     .extern .llvm_gcda_start_file[PR]
+; CHECK-NEXT:    .extern .llvm_gcda_emit_function[PR]
+; CHECK-NEXT:    .extern .llvm_gcda_emit_arcs[PR]
+; CHECK-NEXT:    .extern .llvm_gcda_summary_info[PR]
+; CHECK-NEXT:    .extern .llvm_gcda_end_file[PR]
+; CHECK-RW-NEXT:    .ref __llvm_covinit[RW]
+; CHECK-RO-NEXT:    .ref __llvm_covinit[RO]
```

Comment:
This seems exceedingly weird. Why would we generate a label for `__llvm_gcov_ctr.1:` and not specify the space it needs?

Reply:
Fixed. I think this is because of the global merge that happens in `llc`.

Reply:
> Fixed. I think this is because of the global merge that happens in llc.

Okay. We're seeing an existing bug here then. I'll see about getting an issue opened.